src/frontend/src/routes/(new-styling)/+page.svelte

```diff
@@ -63,89 +61,86 @@
   <div class="flex flex-1 flex-col items-center justify-center">
     <AuthPanel class="sm:max-w-100">
       <div class="flex-1"></div>
-      {#if nonNullish(authFlow.captcha)}
-        <SolveCaptcha {...authFlow.captcha} />
-      {:else}
+      {#if lastUsedIdentities.length > 0}
         <h1 class="text-text-primary my-2 self-start text-2xl font-medium">
```

Comment:
It would be nice to have this as a separate view as well. For readibility purposes mostly.